src/main/java/uk/gov/digital/ho/proving/income/audit/statistics/PassStatisticsCalculator.java

```diff
@@ -49,4 +53,16 @@ private boolean isInDateRange(AuditResultByNino auditResult, LocalDate fromDate,
         return resultsInRange.stream()
             .collect(groupingBy(AuditResultByNino::resultType, Collectors.counting()));
     }
+
+    private long combineCounts(AuditResultType resultType, Map<AuditResultType, Long> countsByResult, List<ArchivedResult> archivedResults) {
```

Comment:
I can't see any tests with countsByResult containing any kind of meaningful data - always empty list.  It looks like it would work but...

Reply:
I missed these out by accident - this was a huge PR that I've broken up. I've added them in now